### PR TITLE
Feature/iops 1626 healthcare service

### PR DESCRIPTION
**Pull request by digitalhealthmartin**

### CODE REVIEW

**Comment by digitalhealthmartin** [comment]
Removed unrelated files

File: StructureDefinition/England-HealthcareService.xml [hunk 1]
```diff
@@ -0,0 +1,307 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+    <id value="England-HealthcareService" />
+    <url value="https://fhir.nhs.uk/StructureDefinition/England-HealthcareService" />
+    <name value="EnglandHealthcareService" />
+	<title value="England HealthcareService" />
+    <status value="draft" />
+    <fhirVersion value="4.0.1" />
+    <kind value="resource" />
+    <abstract value="false" />
+	<version value="1.0.0" />
```
Comment (SBajanki1) [inline StructureDefinition/England-HealthcareService.xml:10]: This looks different to other profiles. Missing  <date value="2023-06-01" />
  <publisher value="NHS England" />
  <contact>
    <name value="NHS England" />
    <telecom>
      <system value="email" />
      <value value="interoperabilityteam@nhs.net" />
      <use value="work" />
      <rank value="1" />
    </telecom>
  </contact>

File: StructureDefinition/England-HealthcareService.xml [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    <type value="HealthcareService" />
```
Comment (SBajanki1) [inline StructureDefinition/England-HealthcareService.xml:11]: This looks different to other profiles. Missing contact details, description and purpose

File: codesystems/CodeSystem-England-ClinicalSpecialty.xml [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+      <name value="NHS England" />
+      <telecom>
+         <system value="email" />
+         <value value="interoperabilityteam@nhs.net" />
+         <use value="work" />
+		 <rank value="1" />
+      </telecom>
+   </contact>
+   <description value="A CodeSystem to identify the specialty of the care professional or team. FHIR Version of NHS Data Dictionary Main Specialty Code and HSCIC ITK HL7v2 Table ZU012" />
+   <copyright value="Copyright © 2019 NHS Digital" />
```
Comment (SBajanki1) [inline codesystems/CodeSystem-England-ClinicalSpecialty.xml:20]: purpose is missing

File: codesystems/CodeSystem-England-ERSAppointmentType.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A Code System for eRS Appointment Type" />
+	<purpose value="A Code System for eRS Appointment Type" />
+    <copyright value="Copyright &#169; 2022 NHS Digital" />
```
Comment (SBajanki1) [inline codesystems/CodeSystem-England-ERSAppointmentType.xml:21]: Update the copyright

File: valusets/ValueSet-England-ERSAppointmentType.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A ValueSet to describe the type of appointment" />
+	<purpose value="A ValueSet to describe the type of appointment" />
+    <copyright value="Copyright &#169; 2022 NHS Digital" />
```
Comment (SBajanki1) [inline valusets/ValueSet-England-ERSAppointmentType.xml:21]: Update copyright

File: valusets/ValueSet-England-ERSClinicType.xml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A valueset to describe the clinic type" />
+	<purpose value="A valueset to describe the clinic type" />
+    <copyright value="Copyright &#169; 2017 NHS Digital" />
```
Comment (SBajanki1) [inline valusets/ValueSet-England-ERSClinicType.xml:20]: Update copyright

File: StructureDefinition/England-HealthcareService.xml (hunk 2 above)
Comment (SheenaSharma) [reply StructureDefinition/England-HealthcareService.xml:11]: Copyright is also missing.

File: codesystems/CodeSystem-England-ERSPurpose.xml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A Code System for eRS Purpose" />
+	<purpose value="A Code System for eRS Purpose" />
+    <copyright value="Copyright &#169; 2022 NHS Digital" />
```
Comment (SheenaSharma) [inline codesystems/CodeSystem-England-ERSPurpose.xml:21]: Copyright needs to be updated.

File: codesystems/CodeSystem-England-ERSRequestFlowType.xml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A Code System for eRS Request Flow Type" />
+	<purpose value="A Code System for eRS Request Flow Type" />
+    <copyright value="Copyright &#169; 2022 NHS Digital" />
```
Comment (SheenaSharma) [inline codesystems/CodeSystem-England-ERSRequestFlowType.xml:21]: Copyright needs to be updated.

File: codesystems/CodeSystem-England-ERSSpecialty.xml [hunk 9]
```diff
@@ -0,0 +1,387 @@
...
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
+        </telecom>
+    </contact>
+    <description value="ERS Specialty type" />
+	<purpose value="ERS Speciality type" />
+    <copyright value="Copyright &#169; 2017 NHS Digital" />
```
Comment (SheenaSharma) [inline codesystems/CodeSystem-England-ERSSpecialty.xml:20]: Copyright needs to be updated.

File: codesystems/CodeSystem-England-ERSSpecialty.xml [hunk 10]
```diff
@@ -0,0 +1,387 @@
...
+	<title value="England ERS Speciality" />
+    <status value="draft" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
```
Comment (SheenaSharma) [inline codesystems/CodeSystem-England-ERSSpecialty.xml:15]: Please align rank value in proper place.

File: valusets/ValueSet-England-ERSAppointmentType.xml [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
```
Comment (SheenaSharma) [inline valusets/ValueSet-England-ERSAppointmentType.xml:16]: Please align rank value.

File: valusets/ValueSet-England-ERSClinicType.xml [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+	<title value="England ERS Clinic Type" />
+    <status value="draft" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
```
Comment (SheenaSharma) [inline valusets/ValueSet-England-ERSClinicType.xml:15]: Please align rank value properly.

File: valusets/ValueSet-England-ERSSpeciality.xml [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+	<title value="England ERS Speciality" />
+    <status value="draft" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
```
Comment (SheenaSharma) [inline valusets/ValueSet-England-ERSSpeciality.xml:15]: Please align rank value.

**Comment by digitalhealthmartin** [comment]
Added CS and made fixes to files re review comments

**Comment by digitalhealthmartin** [comment]
**OLD PR. DO NOT MERGE. CLOSING PR.**

**Pull request closed without merging**